src/humiolib/HumioClient.py

```diff
@@ -641,7 +642,7 @@ def _ingest_messages(
     ):
         """
         Ingest unstructred messages to repository.
-        Structure of ingested data is discussed in: https://docs.humio.com/api/ingest/#structured-data
+        Structure of ingested data is discussed in: https://docs.humio.com/reference/api/ingest/#parser
```

Comment:
I think this should be changed to:
` Ingest of unstructured data is discussed in: https://docs.humio.com/reference/api/ingest/#parser `